fonts/OTF/TeX/makeFF

```diff
@@ -1517,11 +1517,12 @@ $map{cmtt10} = {
     0x15 => [0x306,-525,0], # \breve (combining)
     0x16 => [0x304,-525,0], # \bar (combining)
     0x17 => [0x30A,-525,0], # ring above (combining)
+    0x20 => 0x2423,         # graphic representation of space
```

Comment:
I see.  I was hoping that `cmtt` would have an real space, but that doesn't seem to be the case.